tensorflow_probability/python/distributions/loglogistic.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2018 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""LogNormal distribution classes."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow.compat.v2 as tf
+
+from tensorflow_probability.python.bijectors import exp as exp_bijector
+from tensorflow_probability.python.distributions import logistic
+from tensorflow_probability.python.distributions import transformed_distribution
+from tensorflow_probability.python.internal import assert_util
+from tensorflow_probability.python.internal import dtype_util
+from tensorflow_probability.python.internal import tensor_util
+
+__all__ = [
+    'LogLogistic',
+]
+
+
+class LogLogistic(transformed_distribution.TransformedDistribution):
+  """The log-logistic distribution."""
+
+  def __init__(self,
+               alpha,
```

Comment:
We typically use terms like `scale` and `concentration` for these, and avoid greek.

Reply:
Regarding the greeks, I'll change that, thx